magma/spec/update_spec.rb

```diff
@@ -881,5 +900,23 @@ def update(revisions, user_type=:editor)
       victim.refresh
       expect(victim.country).to eq('thrace')
     end
+
+    it 'allows updates to the restricted column by a privileged user' do
+      victim = create(:victim, name: 'Outis Koutsonadis', country: 'nemea')
+
+      update(
+          {
+              victim: {
+                  'Outis Koutsonadis': {
+                      restricted: false
+                  }
+              }
+          },
+          :editor
+      )
+      expect(last_response.status).to eq(200)
+
+      expect { victim.refresh }.to change { victim.restricted }.to(false)
```

Comment:
nit: By default isn't `victim.restricted` set to `false` on `create`, since it wasn't specified? -- should this be testing the opposite (setting it to `true` on update), or should the create() call set it to `true`?

Reply:
It is set to `nil` actually.  If I start from true, then the call itself would fail for not being able to find the victim, rather than not being able to update its state. The test still works from the perspective of setting it to a value from the nil state.